script/add-payment-partials-to-marriage-abroad-services-data.rb

```diff
@@ -1,9 +1,9 @@
 countries = %w(
-  kuwait
+  laos
 )
 
 payment_method_partial = {
-  'default' => 'pay_by_cash_or_credit_card_no_cheque_nor_amex'
+  'default' => 'pay_by_cash_or_credit_card_no_cheque'
 }
 
 marriage_abroad_services_file = Rails.root.join('lib', 'data', 'marriage_abroad_services.yml')
```

Comment:
I've updated the commit note in PR #2511. See https://github.com/alphagov/smart-answers/pull/2511/commits/e53699ed16b6ae009e4ba7bb6458940b90e202b8.